orchestrator/assets/waste.py

```diff
@@ -113,6 +119,45 @@ def small_stream_recycle(dhub: ResourceParam[DataHubResource]):
     return df_out
 
 
+@asset(
+    io_manager_key="postgres_replace",
+    compute_kind="python",
+    group_name="raw",
+)
+def waste_emission_factors_EPA(dhub: ResourceParam[DataHubResource]):
+    """This asset ingest the Commuting Emission Factors by vehicle
+    type from EPA Emission Factors Hub."""
+    project_id = dhub.get_project_id("Scope3 General")
+    logger.info(f"Found project id: {project_id}!")
+    download_links = dhub.search_files_from_project(project_id, "EPA_GHG_emission_factors_June_2024")
+    if len(download_links) == 0:
+        logger.error("No download links found!")
+        return pd.DataFrame()
+    # Load the data
+    workbook = pd.ExcelFile(download_links[0], engine="openpyxl")
```

Comment:
maybe we could add error handling here in case the file is not found or is corrupted?

Reply:
great thinking, what behavior would u like to see?

```python
    try:
        workbook = pd.ExcelFile(download_links[0], engine="openpyxl")
    except FileNotFoundError:
        logger.error("Not able to load the file")
        return pd.DataFrame()
```

Something like this would work, but might be redundant as we just check if the link is available in the last line?